base/shell/explorer/traywnd.cpp

```diff
@@ -2764,6 +2764,26 @@ class CTrayWindow :
         return 0;
     }
 
+    LRESULT OnSettingsChange(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
+    {
+        // RegenerateUserEnvironment
+        typedef BOOL (WINAPI *REGENERATEUSERENVIRONMENT)(LPVOID *, BOOL);
+
+        HMODULE hShell32 = GetModuleHandleA("shell32");
+
+        REGENERATEUSERENVIRONMENT pRegenerateUserEnvironment;
+        pRegenerateUserEnvironment = (REGENERATEUSERENVIRONMENT)GetProcAddress(hShell32, "RegenerateUserEnvironment");
```

Comment:
explorer links to shell32 so no need to dynamically load RegenerateUserEnvironment.... Does Windows' explorer do that?

Reply:
Directly called RegenerateUserEnvironment.